src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -195,6 +195,9 @@ protected EC2Cloud(String name, boolean useInstanceProfileForCredentials, String
         this.roleSessionName = roleSessionName;
         this.credentialsId = Util.fixEmpty(credentialsId);
         this.sshKeysCredentialsId = Util.fixEmpty(sshKeysCredentialsId);
+        if (this.sshKeysCredentialsId == null) {
+            this.sshKeysCredentialsId = null;
+        }
```

Comment:
Uh, what??

Reply:
PEBKAB

Reply:
i forgot to add the check for `isBlank()` - basically i want to ensure that field is null if there is no possible real value (ie. null or empty)